packages/vitest/src/index.ts

```diff
@@ -26,6 +26,10 @@ declare module 'vite' {
   }
 }
 
+export function defineConfig(config: { test?: VitestInlineConfig }) {
```

Comment:
The type should be `UserConfig` from `vite`

Reply:
Updated